lib/komponent/core/component_path_resolver.rb

```diff
@@ -0,0 +1,66 @@
+module Komponent
+  class ComponentPathResolver
+    attr_accessor :path
+
+    def initialize(component)
+      @component = component
+      resolve
```

Comment:
I think if it's a "resolver" it should be able to resolve multiple times. The API would be something like `ComponentPathResolver.new.resolve(component_name)`